.travis.yml

```diff
@@ -1,21 +1,35 @@
 dist: bionic
 
-language: node_js
-
 node_js:
   - "12"
 
+python:
```

Comment:
I think you might want to move this into the job definition below, to run the python tests on all these pythons. Right now, it is only running on one python, 3.6. Or just pick one...

And skip the install, so travis stops attempting it's default "install" logic, generating this warning:

```
Could not locate requirements.txt. Override the install: key in your .travis.yml to install dependencies.
```

```
    - stage: test
      name: Project Unit Testing
      language: python
      python:
        - "3.6"
        - "3.7"
        - "3.8"
        - "3.9"
      install: skip
      script:
        - make docker/run target=python/test
```

Reply:
Done